### PR TITLE
Update changelog for v3.5.0

### PR DESCRIPTION
**Pull request by khairul-alam-licon**
# Description
Update change logs for v3.5.0 build.

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 ## CHANGELOG
 
-### 3.X.X (2021-XX-XX)
+### 3.5.0 (2021-07-26)
```
Comment (corycaywood) [inline CHANGELOG.md:3]: You also need to add the fix about File Input `accept` types: https://github.com/rakutentech/android-miniapp/pull/332

Comment (khairul-alam-licon) [reply CHANGELOG.md:3]: Yeah, updated.

**Review by Climbatize — APPROVED** [review]
LGTM

**Review by munir-rakuten — APPROVED** [review]
Looks good to me 👍

**Review by corycaywood — APPROVED** [review]

**Pull request merged**